content/en/blog/_posts/2021-03-09-Defining-NetworkPolicy-Conformance-For-CNIs

```diff
@@ -0,0 +1,373 @@
+---
+layout: blog
+title: "Defining NetworkPolicy Conformance for CNI providers"
+date: 2021-03-21
+slug: defining-networkpolicy-conformance-cni-providers
+---
+
+Authors: Matt Fennwick, Jay Vyas, Ricardo Katz, Amim Knabben, Douglas Landgraf
+
+A brief conversation around "node local" network policies in April of 2020 inspired the creation of a NetworkPolicy subproject from sig-network.
+
+In this post we'll discuss:
+
+- Why we created a Subproject for Network Policies 
+- How we changed the Kubernetes e2e framework to `visualize` NetworkPolicy implementation of your CNI provider
```

Comment:
We might also want to expand CNI to “Container Network Interface (CNI) plugin”

Reply:
sure, changed the title and also the first appearance of CNIs